controller/src/scheduler.rs

```diff
@@ -77,6 +77,7 @@ impl Scheduler {
 #[cfg(test)]
 mod tests {
     use super::*;
+    const PLANE_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
Is the CARGO_PKG_VERSION the same as the commit hash?

Reply:
No, it is the version string from the [cargo toml](https://github.com/drifting-in-space/plane/blob/main/drone/Cargo.toml#L3).

I'd like to move away from deploying based on the sha hash, and instead to use numbered versions. I'm not ready to do that yet, but this will help when we do.